tests/src/test/java/alluxio/server/ft/MasterFaultToleranceIntegrationTest.java

```diff
@@ -247,15 +249,23 @@ public void killStandby() throws Exception {
     }
   }
 
+  @Test(timeout = 30000)
+  public void queryStandby() throws Exception {
+    List<InetSocketAddress> addresses = mMultiMasterLocalAlluxioCluster.getMasterAddresses();
+    Collections.shuffle(addresses);
+    PollingMasterInquireClient inquireClient =
+        new PollingMasterInquireClient(addresses, ServerConfiguration.global());
+    assertEquals(mMultiMasterLocalAlluxioCluster.getLocalAlluxioMaster().getAddress(),
+        inquireClient.getPrimaryRpcAddress());
+  }
+
   @Test
   public void workerReRegister() throws Exception {
-    AlluxioBlockStore store =
-        AlluxioBlockStore.create(FileSystemContext.create(ServerConfiguration.global()));
-    assertEquals(WORKER_CAPACITY_BYTES, store.getCapacityBytes());
-
     for (int kills = 0; kills < MASTERS - 1; kills++) {
       assertTrue(mMultiMasterLocalAlluxioCluster.stopLeader());
       mMultiMasterLocalAlluxioCluster.waitForNewMaster(CLUSTER_WAIT_TIMEOUT_MS);
+      AlluxioBlockStore store =
```

Comment:
Does this need to be inside the loop? I moved it outside before so we didn't have to re-create the FileSystemContext each time

Reply:
I'll revert this

Reply:
actually reverting it causes the test to fail because the eligible workers are cached and need to be cleared